spec/swagger/swagger_doc_v1.yaml

```diff
@@ -284,6 +284,48 @@
             application/vnd.api+json:
               schema:
                 "$ref": "../v1/post_allocations_responses.yaml#/422"
+  "/allocations/{allocation_id}":
+    get:
+      summary: Returns the details of an allocation
+      tags:
+        - Allocations
+      consumes:
+        - application/vnd.api+json
+      parameters:
+        - "$ref": "../v1/content_type_parameter.yaml#/ContentType"
+        - "$ref": "../v1/allocation_id_parameter.yaml#/AllocationId"
+        - "$ref": "../v1/allocation_include_parameter.yaml#/AllocationIncludeParameter"
+      responses:
+        "200":
+          description: success
+          content:
+            application/vnd.api+json:
+              schema:
+                "$ref": "../v1/get_allocation_responses.yaml#/200"
```

Comment:
Could you amend `get_allocation_responses.yaml` as its not structured correctly thus when viewing the swagger docs generated it's coming back as `{}` for the 200 response

Reply:
Good catch, well spotted and now resolved 👍